Alignment/OfflineValidation/plugins/TrackerOfflineValidationSummary.cc

```diff
@@ -216,6 +222,10 @@ void TrackerOfflineValidationSummary::endJob() {
   lastSetup_->get<TrackerTopologyRcd>().get(tTopoHandle);
```

Comment:
Hi @mmusich , so this part is left as you don't know how to re-write it?

Reply:
@yuanchao
right, apparently there is no right transition from the framework to choose for the `endJob` (probably because one is not supposed to tamper with the ES at the endJob).
Do you have suggestions?

Reply:
@mmusich Could you try consulting with Christopher Jones (cdj@fnal.gov) or Matti (matti.kortelainen@cern.ch) for some advice here? For other parts I've checked and they look ok.

Reply:
Then please sign this PR, I have more changes on top of this.